source/whats-new.txt

```diff
@@ -58,6 +58,24 @@ What's New in 6.0
      import these methods from the driver instead of from
      ``mongodb-client-encryption``.
 
+   - Removed support for strings with a length of 12 from the ``ObjectId``
+     constructor.
+
+   - The ``abortTransaction`` and ``commitTransaction`` methods return
+     ``null`` instead of the raw command results.
+
+   - Removed connection option helpers that recognized values other than ``true``
```

Comment:
Suggestion:

While this style guide guidance is sometimes borderline, it mentions [don't humanize inanimate objects](https://www.mongodb.com/docs/meta/style-guide/writing/use-effective-verbs/#don-t-humanize-inanimate-objects). I think it is not common to describe methods as performing the "recognize" action:

```suggestion
   - Removed connection option helpers that accepted values other than ``true``
```

Other alternatives could be "parsed" or "casted"